### DISCUSSION TITLE
dynamic route doesn't show the return content as expected

### DISCUSSION THREAD

**Discussion by jonatsuji** [neuefische/questions#393] [Web] [answered]
Hi everyone,

I created a route to a dynamic js-file ([areaSectors].js) in my pages, which is setup to filter an array and take the results for my return content. I hope the screenshots explain it a bit more. 

Anyway, I don't get any error but my page doesn't show any content and stays empty.

can you have a look on this?
I pushed it, so you can also have a look in the PR [here](https://github.com/jonatsuji/capstone-project/pull/2)

Many thanks in advance.
![Screenshot 2022-12-14 at 11 40 17](https://user-images.githubusercontent.com/115465230/207577201-fa4b51bd-3848-4ebe-bdcd-3e06f2eb0a9f.png)
![Screenshot 2022-12-14 at 11 43 23](https://user-images.githubusercontent.com/115465230/207577237-afb852a3-8b63-4ec3-82b0-5bb8a1cb592c.png)

**Comment by actyralf** [comment, marked as answer]
Instead of `slug` you should extract `areaSectors` from `router.query`. This will contain the dynamic part of the url.
One tip: I guess this will be the name of a single sector, so it would be better to rename it to `areaSector` for readability reasons. In this case you would have to rename your file to  `[areaSector].js`

**Reply by jonatsuji** [reply]
This fixed my issue! 
I also renamed as you suggested, thanks for your help!

**Comment by MerleMak** [comment]
On a side note: every directory in the pages folder should be named in lowercase :) so routes and sectors in your case 👍🏼
